### DISCUSSION TITLE
BigWig file tracks are blank

### DISCUSSION THREAD

**Discussion by eliza-naismith** [GMOD/jbrowse-components#4390] [Q&A] [answered]
Hi, I have BigWig files (converted from .wig files) containing the positions of Tn-seq insertions. The files are stored locally on my computer, and I upload them within the desktop app using the 'open tracks' and BigWig adaptor > Quantitative track. However each time I try to upload these into JBrowse the tracks are blank, even though the y-axis has the correct min/max range. Am I doing something wrong?![image](https://github.com/GMOD/jbrowse-components/assets/156012983/91b78809-b728-46e2-873a-0ed9be81ac99) 
<img width="508" alt="image" src="https://github.com/GMOD/jbrowse-components/assets/156012983/4492130d-b473-40ca-8e6e-410a38799e4e">

**Comment by cmdcolin** [comment]
The most common cause of a track being blank is for the "refseq" names to mismatch e.g. the FASTA file says chr1 but the data file says e.g. chrom1. You can use the command "bigWigInfo -chroms volvox-sorted.bam.coverage.bw" using the bigWigInfo command from UCSC (or you can look at your plaintext wig file for the same thing, just search the wig for e.g. LT708304.1)

Random extra note: we're considering adding a built in tool for doing this as well so that you can verify this in the app!

**Reply by cmdcolin** [reply, marked as answer]
Extra extra note: if you can see that the refnames look correct in the file, but you're still not seeing data, that would probably be a weird bug. I'd be interested to see the bigwig file in that case

**Reply by eliza-naismith** [reply]
Thank you for your prompt reply, so helpful as I'm new to using JBrowse! I took a closer look at the refseq names in my original FASTA file and had made an error when generating the bigWig file - the track is working now perfectly. I do like the sound of the built in tool for verifying the names in the app. 😄 
![image](https://github.com/GMOD/jbrowse-components/assets/156012983/2417a102-bbfc-4117-b29b-8fe57d7d97c5)
